app/components/notifier/modal-new-edit/component.js

```diff
@@ -160,4 +161,27 @@ export default Component.extend(ModalBase, NewOrEdit, {
     get(this, 'modalService').toggleModal();
   },
 
+  validate() {
+    this._super();
+    const errors = get(this, 'errors') || [];
+    const intl = get(this, 'intl')
+    const preError = '"Default Recipient" is required'
+
+    if (errors.includes(preError)) {
+      const notifierType = get(this, 'model.notifierType')
+      let afterError = ''
+
+      if (notifierType === 'slack') {
+        afterError = 'Default Channel'
+      }
+      if (notifierType === 'email') {
+        afterError = 'Default Recipient Address'
+      }
+      errors.splice(errors.findIndex((e) => e === preError), 1, intl.t('validation.required', { key: afterError }))
```

Comment:
If `notifierType` is anything other than `email` or `slack` the translated error string will be off because the  `afterError` will not have a translation key string. if the only options for `notifierType` are slack/email use an if/else, if not you'll either need a default string or populate the errors array for each `if` statement so you're not pushing an empty translation key.